### ISSUE TITLE
cf-python cannot open ibtrac netcdf file

### ISSUE THREAD

**Issue by davidhassell** [NCAS-CMS/cf-python#78]
Copied from https://bitbucket.org/cfpython/cf-python/issues/62/cf-python-cannot-open-ibtrac-netcdf-file

An attempt to open this file (https://www.ncei.noaa.gov/data/international-best-track-archive-for-climate-stewardship-ibtracs/v04r00/access/netcdf/IBTrACS.ALL.v04r00.nc)  results in:
```
TypeError: '<' not supported between instances of 'str' and 'int'
```
I’m not sure if it will be recognisable trajectory data when it opens, but that’s a question for another day.

**Comment by davidhassell** [comment]
This is a bug, which will be fixed in the next release of cf-python (due in 7-14 days). Thanks for finding it. In the meantime, you can proceed by hacking the Conventions attribute to be `"CF-1.8”` (it’s currently `”ACDD-1.3”`):

```
$ ncatted -O -a Conventions,global,o,c,"CF-1.8" IBTrACS.ALL.v04r00.nc
$ python
>>> import cf
>>> cf.__version__
'3.4.0'
>>> f = cf.read('IBTrACS.ALL.v04r00.nc')
>>> print(f.select('tropical_cyclone_maximum_sustained_wind_speed')[5])
Field: tropical_cyclone_maximum_sustained_wind_speed (ncvar%usa_wind)
---------------------------------------------------------------------
Data            : tropical_cyclone_maximum_sustained_wind_speed(ncdim%storm(13376), ncdim%date_time(360)) kts
Auxiliary coords: time(ncdim%storm(13376), ncdim%date_time(360)) = [[1842-10-25 03:00:00, ..., --]] standard
                : longitude(ncdim%storm(13376), ncdim%date_time(360)) = [[80.30000305175781, ..., --]] degrees_east
                : latitude(ncdim%storm(13376), ncdim%date_time(360)) = [[10.899999618530273, ..., --]] degrees_north
>>>
```
The bug is related to the parsing of the Conventions global attribute when there are multiple conventions, or the conventions are not CF. The latter case is in play here, and when fixed it will assume the latest conventions if they are not explicitly stated.

David

**Comment by davidhassell** [comment]
Fixed in `cfdm` v1.8.4 and will be inherited by `cf-python`
